python/cuml/cuml/testing/utils.py

```diff
@@ -245,14 +243,6 @@ def sqnorm(x):
     # requirement of minimality for core points
 
 
-def get_handle(use_handle, n_streams=0):
```

Comment:
Some of our tests were parametrized to run both with and without a handle specified. These tests passed with the changes here (if I configured the `FutureWarning` for deprecation not to fail). I opted to rip them out instead though since they didn't provide a meaningful benefit and would go away once the deprecation is completed.